Pear/Cells/CommunityUserTableViewCell.swift

```diff
@@ -106,12 +106,17 @@ class CommunityUserTableViewCell: UITableViewCell {
                 interests = userInterests
                 interestsCollectionView.reloadData()
             }
-             if let profilePictureURL = user.profilePictureURL {
-                 profileImageView.kf.setImage(with: Base64ImageDataProvider(base64String: profilePictureURL, cacheKey: user.netID))
-             }
+            if let profilePictureURL = URL(string: user.profilePictureURL ?? "") {
+                profileImageView.kf.setImage(with: profilePictureURL)
```

Comment:
I'm not too sure, but would ```if let urlString = user.profilePictureURL, let profilePictureURL = URL(string: urlString) { ... } ``` work?

Reply:
Ya, this can work -- this is kind of a filler code for now since gonzalo wiped photos from every user. eventually, im planning to make `user.profilePictureURL` a non-optional so this shouldnt be a problem anymore.